{"0.1.0":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.0/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.0","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"eb1c7274298ea26dbef3985705c4382bd94face3cc87f150bd02a142b2a3f8f5","md5":"5f5bf4c7158cebd0bc86f1cbd9aa036b","sha256":"1bfc71cb56a450adb49c035da8dac2c194c5e1704a89875251a0ff6ab4cb1937"},"downloads":-1,"filename":"llm_error_handler-0.1.0-py3-none-any.whl","has_sig":false,"md5_digest":"5f5bf4c7158cebd0bc86f1cbd9aa036b","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":3376,"upload_time":"2024-06-12T12:29:20","upload_time_iso_8601":"2024-06-12T12:29:20.053607Z","url":"https://files.pythonhosted.org/packages/eb/1c/7274298ea26dbef3985705c4382bd94face3cc87f150bd02a142b2a3f8f5/llm_error_handler-0.1.0-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"8fe0395329da05cb09c58d8ce457df899ee4499fc21f077c433ada9cfb1918b7","md5":"b6650b56e4700cfc3e09875f821f10ca","sha256":"209424c1a825c5fced4ee2646dc3e38b9c2ecf974e8d128089182ba77156b52b"},"downloads":-1,"filename":"llm_error_handler-0.1.0.tar.gz","has_sig":false,"md5_digest":"b6650b56e4700cfc3e09875f821f10ca","packagetype":"sdist","python_version":"source","requires_python":">=3.6","size":3048,"upload_time":"2024-06-12T12:29:21","upload_time_iso_8601":"2024-06-12T12:29:21.983834Z","url":"https://files.pythonhosted.org/packages/8f/e0/395329da05cb09c58d8ce457df899ee4499fc21f077c433ada9cfb1918b7/llm_error_handler-0.1.0.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.1":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.1/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.1","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"fd82110e28f2dccc8b25f4485dc119da4b63895333e4558e700626e7a4a18cde","md5":"f0e28e85727aadecd9ad487c6c274769","sha256":"f9a499f9ed7d07229b2cb34194a81e6bf79af78e7a3cf0f69c5fc8d99e4deb89"},"downloads":-1,"filename":"llm_error_handler-0.1.1-py3-none-any.whl","has_sig":false,"md5_digest":"f0e28e85727aadecd9ad487c6c274769","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":3414,"upload_time":"2024-06-12T12:40:14","upload_time_iso_8601":"2024-06-12T12:40:14.037239Z","url":"https://files.pythonhosted.org/packages/fd/82/110e28f2dccc8b25f4485dc119da4b63895333e4558e700626e7a4a18cde/llm_error_handler-0.1.1-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"2220bfaecee9531be2fcf04d715928315ce3f2a7fba71fb65767d6f6ba70dea5","md5":"4c60b527e6a25c4c6831ecd92c12c086","sha256":"96614d45f40895e27de6d02929965c8045f49d19536784ae40346c02bf620361"},"downloads":-1,"filename":"llm_error_handler-0.1.1.tar.gz","has_sig":false,"md5_digest":"4c60b527e6a25c4c6831ecd92c12c086","packagetype":"sdist","python_version":"source","requires_python":">=3.6","size":3059,"upload_time":"2024-06-12T12:40:16","upload_time_iso_8601":"2024-06-12T12:40:16.133724Z","url":"https://files.pythonhosted.org/packages/22/20/bfaecee9531be2fcf04d715928315ce3f2a7fba71fb65767d6f6ba70dea5/llm_error_handler-0.1.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.10":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.10/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.10","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"4de72509e9d3116f369bad95d62cfe618ad679ce31fa9a687bc8ed827b947c57","md5":"b850d4a995e68842a1d187b2f87833ae","sha256":"c9de32ae9f5ae35d2305a5d3d4d5e92072f99ff13ef7c3f8bc7ecbd9a75fcaa7"},"downloads":-1,"filename":"llm_error_handler-0.1.10-py3-none-any.whl","has_sig":false,"md5_digest":"b850d4a995e68842a1d187b2f87833ae","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":29751,"upload_time":"2024-06-12T13:40:15","upload_time_iso_8601":"2024-06-12T13:40:15.814511Z","url":"https://files.pythonhosted.org/packages/4d/e7/2509e9d3116f369bad95d62cfe618ad679ce31fa9a687bc8ed827b947c57/llm_error_handler-0.1.10-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.11":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.11/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.11","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"4677fa5d038a70aca3b92d11edae4488f5a8d4bb396f135cf4907c77cf5768f1","md5":"56074619b4cc562c6d5c9589293f3321","sha256":"3bcf803193179d83f331564b476bf863988b8f9222f5e204f71be8b79b758971"},"downloads":-1,"filename":"llm_error_handler-0.1.11-py3-none-any.whl","has_sig":false,"md5_digest":"56074619b4cc562c6d5c9589293f3321","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":29768,"upload_time":"2024-06-12T13:44:53","upload_time_iso_8601":"2024-06-12T13:44:53.874898Z","url":"https://files.pythonhosted.org/packages/46/77/fa5d038a70aca3b92d11edae4488f5a8d4bb396f135cf4907c77cf5768f1/llm_error_handler-0.1.11-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.12":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.12/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.12","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"9b7ec814ea767d5748c89375e4242055a3de1b29a9e992a34dd84a9e0175baa8","md5":"64955cc53abc59817f5626d68fa60c06","sha256":"baabe7bc5851b8e3a5e09f8e0110d972017a0456f08797deb1cb195a6e5670ef"},"downloads":-1,"filename":"llm_error_handler-0.1.12-py3-none-any.whl","has_sig":false,"md5_digest":"64955cc53abc59817f5626d68fa60c06","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":29834,"upload_time":"2024-06-12T13:48:30","upload_time_iso_8601":"2024-06-12T13:48:30.135519Z","url":"https://files.pythonhosted.org/packages/9b/7e/c814ea767d5748c89375e4242055a3de1b29a9e992a34dd84a9e0175baa8/llm_error_handler-0.1.12-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.13":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.13/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.13","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"23be240c202f8a3bb098d532feca059f1c58e1ea8198c8d8c22d22bae80eb639","md5":"9c681386f37add58745bb76a86b2a921","sha256":"3b637f4cf028c28a74f3cde71d534a4987f64ad7db7e5d273d3b593955abd1cf"},"downloads":-1,"filename":"llm_error_handler-0.1.13-py3-none-any.whl","has_sig":false,"md5_digest":"9c681386f37add58745bb76a86b2a921","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":30330,"upload_time":"2024-06-12T13:50:14","upload_time_iso_8601":"2024-06-12T13:50:14.301415Z","url":"https://files.pythonhosted.org/packages/23/be/240c202f8a3bb098d532feca059f1c58e1ea8198c8d8c22d22bae80eb639/llm_error_handler-0.1.13-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.14":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.14/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.14","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"0d3e4a3ad8d37e043ed26b783ec9147870d747cd8500357149c030ea15d41df7","md5":"ab48544783297e7b541598e1b6073c92","sha256":"5b2617b431ed4397b6cf2e6bc7fbbf8697e108a0813f9b940b320754f5367342"},"downloads":-1,"filename":"llm_error_handler-0.1.14-py3-none-any.whl","has_sig":false,"md5_digest":"ab48544783297e7b541598e1b6073c92","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":30331,"upload_time":"2024-06-12T13:56:39","upload_time_iso_8601":"2024-06-12T13:56:39.305103Z","url":"https://files.pythonhosted.org/packages/0d/3e/4a3ad8d37e043ed26b783ec9147870d747cd8500357149c030ea15d41df7/llm_error_handler-0.1.14-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.15":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.15/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.15","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"19b4a3345e6595ed051d0b21d6c20406bd18dc87fdbd240c387204033659e58a","md5":"5cc046b056e5cb38efda2fd75a8dc1e9","sha256":"a1c4c527473d62123efd1e94dbe8628e786be50596112fd1a2cc9557df34c0a0"},"downloads":-1,"filename":"llm_error_handler-0.1.15-py3-none-any.whl","has_sig":false,"md5_digest":"5cc046b056e5cb38efda2fd75a8dc1e9","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":30462,"upload_time":"2024-06-12T14:05:15","upload_time_iso_8601":"2024-06-12T14:05:15.051269Z","url":"https://files.pythonhosted.org/packages/19/b4/a3345e6595ed051d0b21d6c20406bd18dc87fdbd240c387204033659e58a/llm_error_handler-0.1.15-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.5":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.5/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.5","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"31cff02622c6644636a3ca917e06882d0050390965d2148f8aa6d15ab0e7c421","md5":"cd5de767a1e1ccc526531ada58673b70","sha256":"766eaf5c3ab64a1c6492c5c391e0c82cfe5c9d0130e33472d490e301cfb4b041"},"downloads":-1,"filename":"llm_error_handler-0.1.5-py3-none-any.whl","has_sig":false,"md5_digest":"cd5de767a1e1ccc526531ada58673b70","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":3414,"upload_time":"2024-06-12T13:09:22","upload_time_iso_8601":"2024-06-12T13:09:22.832405Z","url":"https://files.pythonhosted.org/packages/31/cf/f02622c6644636a3ca917e06882d0050390965d2148f8aa6d15ab0e7c421/llm_error_handler-0.1.5-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.7":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.7/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.7","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"a93e18709592a1c23252e1ecea496e7faccd6dcb4df8486d4229b343ec29de3e","md5":"7865808c335baaac0cbfddd8b57125c9","sha256":"7c2dd307d1af3400822832eadd299060c89bd36e92475f704737d576c9eb2d31"},"downloads":-1,"filename":"llm_error_handler-0.1.7-py3-none-any.whl","has_sig":false,"md5_digest":"7865808c335baaac0cbfddd8b57125c9","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":3913,"upload_time":"2024-06-12T13:21:10","upload_time_iso_8601":"2024-06-12T13:21:10.846837Z","url":"https://files.pythonhosted.org/packages/a9/3e/18709592a1c23252e1ecea496e7faccd6dcb4df8486d4229b343ec29de3e/llm_error_handler-0.1.7-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.8":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.8/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.8","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"b8263daa327ea72130c6aac4603676192016e528402133f01431a2312763d138","md5":"7a1b00b882db6ce4521c3160a5eaff26","sha256":"49b1b0ba082d2bef2d374e356bd0f68e947617a9ef82cb9cae9764e67d0764b3"},"downloads":-1,"filename":"llm_error_handler-0.1.8-py3-none-any.whl","has_sig":false,"md5_digest":"7a1b00b882db6ce4521c3160a5eaff26","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":29392,"upload_time":"2024-06-12T13:30:15","upload_time_iso_8601":"2024-06-12T13:30:15.601334Z","url":"https://files.pythonhosted.org/packages/b8/26/3daa327ea72130c6aac4603676192016e528402133f01431a2312763d138/llm_error_handler-0.1.8-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.1.9":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.1.9/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.1.9","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"c5e50a6b780e2a8452012725467383a8f0451e10cd80f2edb7ccf8d58ed13f76","md5":"3f40c3190bd9f462ccdf2367851d2be4","sha256":"5bae3c6c5aba90034172b0959c47ad45b560da13f6200ceced343a1196a23cdb"},"downloads":-1,"filename":"llm_error_handler-0.1.9-py3-none-any.whl","has_sig":false,"md5_digest":"3f40c3190bd9f462ccdf2367851d2be4","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":29427,"upload_time":"2024-06-12T13:33:53","upload_time_iso_8601":"2024-06-12T13:33:53.389356Z","url":"https://files.pythonhosted.org/packages/c5/e5/0a6b780e2a8452012725467383a8f0451e10cd80f2edb7ccf8d58ed13f76/llm_error_handler-0.1.9-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]},"0.2.15":{"info":{"author":"Shoichi","author_email":"s.kitano.h050521@gmail.com","bugtrack_url":null,"classifiers":["License :: OSI Approved :: MIT License","Operating System :: OS Independent","Programming Language :: Python :: 3"],"description_content_type":"text/markdown","docs_url":null,"download_url":null,"downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/shokitano0521/LLM-Error-Handler","keywords":null,"license":"MIT","maintainer":null,"maintainer_email":null,"name":"llm-error-handler","package_url":"https://pypi.org/project/llm-error-handler/","platform":null,"project_url":"https://pypi.org/project/llm-error-handler/","project_urls":{"Homepage":"https://github.com/shokitano0521/LLM-Error-Handler"},"provides_extra":null,"release_url":"https://pypi.org/project/llm-error-handler/0.2.15/","requires_dist":["annotated-types==0.7.0","anyio==4.4.0","cachetools==5.3.3","certifi==2024.6.2","charset-normalizer==3.3.2","distro==1.9.0","exceptiongroup==1.2.1","google-ai-generativelanguage==0.6.4","google-api-core==2.19.0","google-api-python-client==2.133.0","google-auth-httplib2==0.2.0","google-auth==2.30.0","google-generativeai==0.6.0","googleapis-common-protos==1.63.1","grpcio-status==1.62.2","grpcio==1.64.1","h11==0.14.0","httpcore==1.0.5","httplib2==0.22.0","httpx==0.27.0","idna==3.7","numpy==1.26.4","openai==1.30.1","pillow==10.3.0","proto-plus==1.23.0","protobuf==4.25.3","pyasn1==0.6.0","pyasn1-modules==0.4.0","pydantic==2.7.3","pydantic-core==2.18.4","pyparsing==3.1.2","regex==2024.5.15","requests==2.32.3","rsa==4.9","sniffio==1.3.1","tiktoken==0.7.0","tqdm==4.66.4","typing-extensions==4.12.2","uritemplate==4.1.1","urllib3==2.2.1"],"requires_python":">=3.6","summary":"A simple decorator for AI error handling in Python.","version":"0.2.15","yanked":false,"yanked_reason":null},"last_serial":23649358,"urls":[{"comment_text":"","digests":{"blake2b_256":"4c156c7098843768d202a461332cd6cce1eb7d97ddf0bbbe606cd06b84ba05af","md5":"00913a74e77068be963545aa84382b56","sha256":"0a29b338d4d3ca21e09582b64d5b2cff7f564a605e0c87ca7b3097b25429e6b5"},"downloads":-1,"filename":"llm_error_handler-0.2.15-py3-none-any.whl","has_sig":false,"md5_digest":"00913a74e77068be963545aa84382b56","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.6","size":30623,"upload_time":"2024-06-12T23:01:40","upload_time_iso_8601":"2024-06-12T23:01:40.355514Z","url":"https://files.pythonhosted.org/packages/4c/15/6c7098843768d202a461332cd6cce1eb7d97ddf0bbbe606cd06b84ba05af/llm_error_handler-0.2.15-py3-none-any.whl","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}